doc/howto/cross_compiling/cross_compiling_for_android.md

```diff
@@ -0,0 +1,153 @@
+# Build PaddlePaddle for Android
+
+There are two approaches to build PaddlePaddle for Android:
+
+1. Use a Docker container, or
+2. cross-compiling on Linux.
+
+## Docker-based Cross-Compiling
+
+Docker-based cross-compiling is the recommended approach because Docker runs on all major operating systems, including Linux, Mac OS X, and Windows.
+
+### Build the Docker Image
+
+The following steps pack tools we need to build PaddlePaddle into a Docker image.
```

Comment:
Rephrase: The following steps pack all the tools that we need to build PaddlePaddle, into a Docker image.

Reply:
Done